tuna/miopen/subcmd/update_golden.py

```diff
@@ -244,6 +244,8 @@ def actuator2(func):
       session_retry(session, gold_session_update, functools.partial(actuator2),
                     logger)
 
+  LOGGER.info('Finished Updating conv_golden %s', args.golden_v)
```

Comment:
LOGGER should be lower case letters aligned with function args
Something like this:
`logger.info('Finished Updating conv_golden %s', args.golden_v)`